cmake/FindOpenVDB.cmake

```diff
@@ -224,6 +287,28 @@ list(APPEND _OPENVDB_LIBRARYDIR_SEARCH_DIRS
 
 set(_OPENVDB_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
 
+set(OPENVDB_PYTHON_PATH_SUFFIXES
+  ${CMAKE_INSTALL_LIBDIR}/python
+  ${CMAKE_INSTALL_LIBDIR}/python2.7
+  ${CMAKE_INSTALL_LIBDIR}/python3
+  lib64/python
+  lib64/python2.7
+  lib64/python3
+  lib/python
+  lib/python2.7
+  lib/python3
```

Comment:
In theory a user could have installed multiple versions of OpenVDB - say one against Python2 and one against Python3. Is it worth providing a mechanism for them to pick the specific paths they want here?

Reply:
Yeah the variable ` ${OPENVDB_${COMPONENT}_ROOT}` and ` ${OPENVDB_${COMPONENT}_LIBRARYDIR}` should do what you're suggesting here. i.e:
`-DOPENVDB_pyopenvdb_ROOT="/path/to/custom/python/install"`